app/src/main/java/eu/kanade/tachiyomi/ui/reader/ReaderPresenter.kt

```diff
@@ -500,6 +506,44 @@ class ReaderPresenter(
         }
     }
 
+    private fun downloadNextChapters() {
+        val manga = manga ?: return
+        val chaptersNumberToDownload = preferences.autoDownloadAfterReading().get()
+        if (chaptersNumberToDownload == 0 || !manga.favorite) return
+        val isChapterDownloaded = downloadManager.isChapterDownloaded(getCurrentChapter()!!.chapter, manga)
+        if (isChapterDownloaded) {
```

Comment:
so with that parameter passed
```suggestion
        if (isChapterDownloaded || isPrevChapterDownloaded) {
```